GameOfLife/GameOfLifeProcessor/BoardHelper/BoardHelperService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using GameOfLifeProcessor.GameBoard;
+
+namespace GameOfLifeProcessor.BoardHelper
+{
+    internal class BoardHelperService : IBoardHelperService
```

Comment:
der Name Helper deutet meistens schon als CodeSmell an, dass hier ein Verstoß gegen Single Responsibility vorliegen könnte. Oder es in Zukunft zu einem Verstoß kommen wird.

Es ist derzeit nur eine Methode drin.
Wird der Code noch an mehreren Stelle benötigt?
Wird der Code einzelnd benötigt?
Wenn nicht, kann er eigentlich genau dort hin, wo er benutzt wird.
Für mich gehört es zum Game, da wo auch NextGeneration ermittelt wird.